bodhi/tests/client/test___init__.py

```diff
@@ -392,6 +392,6 @@ def test_url_flag(self, send_request, query):
                 'close_bugs': True, 'stable_karma': None, 'csrf_token': 'a_csrf_token',
                 'staging': False, 'builds': u'nodejs-grunt-wrap-0.3.0-2.fc25', 'autokarma': True,
                 'edited': u'nodejs-grunt-wrap-0.3.0-2.fc25', 'suggest': None,
-                'notes': u'this is an edited note', 'request': None, 'bugs': u'',
-                'unstable_karma': None, 'type': 'bugfix'})
+                'notes': u'this is an edited note', 'notes_file': None,
+                'request': None, 'bugs': u'', 'unstable_karma': None, 'type': 'bugfix'})
         self.assertEqual(bindings_client.base_url, 'http://localhost:6543/')
```

Comment:
Oh, and to make it clear - I do think copying @cverna's test and modifying it to use the --notes-file instead of --notes is a good way to do this. That's the way I'd go ☺